markdown.dtx

```diff
@@ -27621,6 +27967,23 @@ end
     \addto@hook\markdownLaTeXTable{#1\\}%
     \expandafter\@gobble
   \fi\markdownLaTeXRenderTableCell}
+%    \end{macrocode}
+% \par
+% \begin{markdown}
+%
+%#### Line blocks
+% Here is a basic implementation of line blocks. If the \pkg{verse} package is
+% loaded, then it is used to produce the verses.
+%
+% \end{markdown}
+%  \begin{macrocode}
+
+\RequirePackage{verse}
```

Comment:
The CI issue [seems related][1] to the verse package.

 [1]: https://github.com/Witiko/markdown/actions/runs/3914794274/jobs/6692285033#step:3:3533

Reply:
Additionally, the verse package should be documented in [the LaTeX soft prerequisites][1].

 [1]: https://github.com/Witiko/markdown/blob/31d90e65a97bccef616a81b33b46b417e212cdd7/markdown.dtx#L1196-L1278

Reply:
Furthermore, we may want to only load the verse package when the `lineBlocks` option is enabled, see for example [strike-through][1].

 [1]: https://github.com/Witiko/markdown/blob/91b8caf102e76649da7f5391087853548750711e/markdown.dtx#L27731-L27753